blog/2023-10-28-hacktoberfest/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Uptane Joins the Open Source Celebration: Hacktoberfest 2023 🎉"
+authors:
+  - Phil
+---
+
+![](hf10_banner_1032x600.png)
+As the leaves change color 🍂 and the breeze carries a hint of chill, the open-source community gears up for a month of code, collaboration, and coffee-fueled late nights ☕. [Hacktoberfest](https://hacktoberfest.com/), the annual open-source celebration, is here again, and this year, Uptane is joining the fray!
+
+Uptane, a robust framework designed to secure software updates for automobiles 🚗, is participating in Hacktoberfest 2023 to foster community engagement and encourage contributions to its project. With cybersecurity becoming a paramount concern in the automotive industry, the initiative taken by Uptane is a timely and significant one.
+
+This Hacktoberfest, developers from around the globe 🌍 can contribute to Uptane's repository, helping to enhance the security framework and make our vehicles more secure. Whether you're a seasoned developer or a newbie looking to make your first open-source contribution, Uptane welcomes you with open arms 🤗.
```

Comment:
One small fix here. Take out the word "our" in the first sentence. Uptane doesn't create or own the vehicles.
```suggestion
This Hacktoberfest, developers from around the globe 🌍 can contribute to Uptane's repository, helping to enhance the security framework and make vehicles more secure. Whether you're a seasoned developer or a newbie looking to make your first open-source contribution, Uptane welcomes you with open arms 🤗.
```